openslides_backend/services/datastore/adapter.py

```diff
@@ -63,6 +63,7 @@ def retrieve(self, command: commands.Command) -> Any:
                 payload = json.loads(content)
             except JSONDecodeError:
                 error_message = "Bad response from datastore service. Body does not contain valid JSON."
+                self.logger.debug(error_message + f" Received: {str(content)}")
```

Comment:
This can be logged as an error, since the datastore should always send valid JSON.

Reply:
Done.